build/makelib/golang.mk

```diff
@@ -177,6 +183,12 @@ $(GO_TOOLS_DIR)/glide:
 	@mv $(GO_TOOLS_DIR)/$(GOHOSTOS)-$(GOHOSTARCH)/glide $(GO_TOOLS_DIR)/glide
 	@rm -r $(GO_TOOLS_DIR)/$(GOHOSTOS)-$(GOHOSTARCH)
 
+.PHONY: $(GO_TOOLS_DIR)/golint
```

Comment:
I added this because without this, it was not overrriding golint if it existed. `make` was saying that everything was up to date, when it wasnt. Particularly if golint existed but it was compiled on a MAC (ie Darwin OS)